R/report.lm.R

```diff
@@ -516,10 +516,10 @@ report_text.lm <- function(x, table = NULL, ...) {
   params <- report_parameters(x, table = table, include_intercept = FALSE, ...)
   table <- attributes(params)$table
 
-  info <- report_info(x, effectsize = attributes(params)$effectsize, parameters = params, ...)
-  model <- report_model(x, table = table, ...)
-  perf <- report_performance(x, table = table, ...)
-  intercept <- report_intercept(x, table = table, ...)
```

Comment:
Are you sure we can remove the dots here?

Reply:
ah no, my bad, I removed them when I ran the code line by line. I'll clean this tomorrow

Reply:
fixed